src/main/java/com/marginallyclever/communications/TransportLayer.java

```diff
@@ -11,17 +13,14 @@
 	 * 
 	 * @return english name of this transport layer
 	 */
-  public String getName();
+    String getName();
```

Comment:
no explicit access rights?

Reply:
This method belongs to an interface which is `public`. You can't reduce the visibility of a method, and as `public` is the highest visibility, you can omit this declaration.

Reply:
i still prefer explicit so there's no doubt.

Reply:
OK reverted